packages/bls12-381/bls12-381.6.0.1/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: """\
+Implementation of BLS12-381 and some cryptographic primitives built
+on top of it"""
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage:
+  "https://nomadic-labs.gitlab.io/cryptography/ocaml-bls12-381/bls12-381/"
+bug-reports:
+  "https://gitlab.com/nomadic-labs/cryptography/ocaml-bls12-381/-/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "ocaml" {>= "4.08"}
+  "ff-sig" {>= "0.6.1" & < "0.7.0"}
+  "zarith" {>= "1.10" & < "2.0"}
+  "zarith_stubs_js" {with-test}
+  "hex" {>= "1.3.0"}
+  "alcotest" {with-test}
+  "integers"
+  "integers_stubs_js" {with-test}
+  "ff-pbt" {>= "0.6.0" & < "0.7.0"}
+  "odoc" {with-doc}
+]
+available: arch != "ppc64" & arch != "arm32" & arch != "x86_32"
```

Comment:
No, it is because 32 bits architectures are not supported, see [here](https://gitlab.com/nomadic-labs/cryptography/ocaml-bls12-381/-/blob/master/src/bindings/blst_bindings_stubs.c#L21)

Reply:
What about ppc64 then?

Reply:
```suggestion
available: arch != "arm32" & arch != "x86_32"
conflicts: [
  "ocaml-option-bytecode-only"
]
```

Reply:
It might be compatible. I have never thought about adding it in the project CI or have seen any user requested ppc64 support. We can see if it works with this CI.